config/src/config/api_config.rs

```diff
@@ -45,6 +51,9 @@ impl Default for ApiConfig {
             tls_key_path: None,
             content_length_limit: None,
             failpoints_enabled: default_disabled(),
+            encode_submission_enabled: default_enabled(),
+            transaction_submission_enabled: default_enabled(),
+            transaction_simulation_enabled: default_enabled(),
```

Comment:
I thought the plan was to disable support for json queries? and maybe bcs?

Reply:
Yeah, I guess I forgot about those.  I can look at that.